src/DfpUrlGenerator.js

```diff
@@ -48,10 +49,10 @@ var dfpUrlGenerator = function () {
 	function buildUrl(obj) {
 		return (obj.protocol || 'http') + '://' +
 			   (obj.host ||
```

Comment:
I think I may be useful to comment out the ES6 and add a comment saying we are converting from ES6 to ES5 format.  That way if we ever go back to ES6 - we do not have to recreate the code.

Reply:
Added explanation comment in this file.